adm_program/system/classes/email.php

```diff
@@ -280,11 +280,7 @@ public function setText($text)
     {
         // Erst mal die Zeilenumbrueche innerhalb des Mailtextes umwandeln in einfache Umbrueche
         // statt \r und \r\n nur noch \n
-        $replaceArray = array(
-            "\r"   => "\n",
```

Comment:
@deki could you test it if it also works if you change line 284 and 285? I think this only is the wrong order

Reply:
You are right but shouldn't we use the same fix for all occurences? That's why I adapted the fix that was applied to systemmail.php in #606.

Reply:
I also don't understand why we must replace `\r`. 
I know `\n` for Linux and `\r\n` for Windows.

Reply:
hmmm... right. why have we done this?